src/SourceBuild/content/build.sh

```diff
@@ -214,7 +214,7 @@ targets="/t:Build"
 if [[ "$test" == true ]]; then
   project="$scriptroot/test/tests.proj"
   targets="$targets;VSTest"
-  properties+=( "/p:_IsTesting=true" )
+  properties+=( "/p:Test=true" )
```

Comment:
this is a very generic name and hard to grep for, what was wrong with the previous name?

Reply:
ah nevermind, I see this is the common arcade property name.